dev/tasks/linux-packages/debian.ubuntu-trusty/control

```diff
@@ -42,6 +44,30 @@ Description: Apache Arrow is a data processing library for analysis
  .
  This package provides header files.
 
+Package: libparquet11
+Section: libs
+Architecture: any
+Multi-Arch: same
+Pre-Depends: ${misc:Pre-Depends}
+Depends:
+  ${misc:Depends},
+  ${shlibs:Depends}
+Description: Apache Parquet is a columner storage format
```

Comment:
There is a typo `columnAr`.

We might want to highlight, that it's the cpp implementation.
> C++ library to read and write Apache Parquet columnar storage format

Reply:
Oh! Thanks for fixing the typo.

C++ information will be useful. We'll add it when we add packages for GLib bindings.